cudapoa/src/cudapoa_kernels.cuh

```diff
@@ -119,40 +126,43 @@ __global__ void generatePOAKernel(uint8_t* consensus_d,
         return;
 
     // Find the buffer offsets for each thread within the global memory buffers.
-    uint8_t* nodes                        = &nodes_d[max_nodes_per_window * window_idx];
-    SizeT* incoming_edges                 = &incoming_edges_d[window_idx * max_nodes_per_window * CUDAPOA_MAX_NODE_EDGES];
-    uint16_t* incoming_edge_count         = &incoming_edge_count_d[window_idx * max_nodes_per_window];
-    SizeT* outoing_edges                  = &outgoing_edges_d[window_idx * max_nodes_per_window * CUDAPOA_MAX_NODE_EDGES];
-    uint16_t* outgoing_edge_count         = &outgoing_edge_count_d[window_idx * max_nodes_per_window];
-    uint16_t* incoming_edge_weights       = &incoming_edge_w_d[window_idx * max_nodes_per_window * CUDAPOA_MAX_NODE_EDGES];
-    uint16_t* outgoing_edge_weights       = &outgoing_edge_w_d[window_idx * max_nodes_per_window * CUDAPOA_MAX_NODE_EDGES];
-    SizeT* sorted_poa                     = &sorted_poa_d[window_idx * max_nodes_per_window];
-    SizeT* node_id_to_pos                 = &node_id_to_pos_d[window_idx * max_nodes_per_window];
-    SizeT* node_alignments                = &node_alignments_d[window_idx * max_nodes_per_window * CUDAPOA_MAX_NODE_ALIGNMENTS];
-    uint16_t* node_alignment_count        = &node_alignment_count_d[window_idx * max_nodes_per_window];
-    uint16_t* sorted_poa_local_edge_count = &sorted_poa_local_edge_count_d[window_idx * max_nodes_per_window];
+    uint8_t* nodes                        = &nodes_d[max_nodes_per_graph * window_idx];
+    SizeT* incoming_edges                 = &incoming_edges_d[window_idx * max_nodes_per_graph * CUDAPOA_MAX_NODE_EDGES];
+    uint16_t* incoming_edge_count         = &incoming_edge_count_d[window_idx * max_nodes_per_graph];
+    SizeT* outgoing_edges                 = &outgoing_edges_d[window_idx * max_nodes_per_graph * CUDAPOA_MAX_NODE_EDGES];
+    uint16_t* outgoing_edge_count         = &outgoing_edge_count_d[window_idx * max_nodes_per_graph];
+    uint16_t* incoming_edge_weights       = &incoming_edge_w_d[window_idx * max_nodes_per_graph * CUDAPOA_MAX_NODE_EDGES];
+    uint16_t* outgoing_edge_weights       = &outgoing_edge_w_d[window_idx * max_nodes_per_graph * CUDAPOA_MAX_NODE_EDGES];
+    SizeT* sorted_poa                     = &sorted_poa_d[window_idx * max_nodes_per_graph];
+    SizeT* node_id_to_pos                 = &node_id_to_pos_d[window_idx * max_nodes_per_graph];
+    SizeT* node_alignments                = &node_alignments_d[window_idx * max_nodes_per_graph * CUDAPOA_MAX_NODE_ALIGNMENTS];
+    uint16_t* node_alignment_count        = &node_alignment_count_d[window_idx * max_nodes_per_graph];
+    uint16_t* sorted_poa_local_edge_count = &sorted_poa_local_edge_count_d[window_idx * max_nodes_per_graph];
 
     int32_t scores_width = window_details_d[window_idx].scores_width;
 
     int64_t scores_offset;
-    if (cuda_banded_alignment)
+    int64_t banded_score_matrix_size;
+    if (banded_alignment || adaptive_banded)
```

Comment:
Optional - can `banded_alignment` variable be renamed to `static_banded` to match naming style of `adaptive_banded`?

Reply:
sure, done.